Tribler/community/tunnel/tunnel_community.py

```diff
@@ -758,6 +800,30 @@ def _ours_on_created_extended(self, circuit, message):
             from Tribler.Core.simpledefs import NTFY_TUNNEL, NTFY_CREATED, NTFY_EXTENDED
             self.notifier.notify(NTFY_TUNNEL, NTFY_CREATED if len(circuit.hops) == 1 else NTFY_EXTENDED, circuit)
 
+
+    def on_introduction_request(self, messages):
+        connectable = False # Todo. Unused so far
```

Comment:
There is actually a connectable flag implemented in Dispersy, each walkcandidate reports its connectability.
See https://github.com/Tribler/dispersy/blob/devel/candidate.py#L130